ushahidi/platform-client
# NEW: Angular Date Picker

This pull request makes the following changes:
- Adds https://github.com/alongubkin/angular-datepicker

Test these changes by:
- click "Filters" in toolbar
- scroll to date fields
- click "start date" field
- date picker should slide from bottom
- interact with date picker (pick a date, clear, close etc...)
- repeat for "end date"

Fixes ushahidi/platform#1078

Ping @ushahidi/platform


---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform-client/205)
